srcpkgs/zeek/patches/fix-musl.patch

```diff
@@ -0,0 +1,23 @@
+--- CMakeLists.txt.orig
++++ CMakeLists.txt
+@@ -322,18 +322,11 @@
+ # if one specifies --with-openssl (which may be common).
+ include_directories(BEFORE ${OPENSSL_INCLUDE_DIR})
+ 
+-# Alpine support
+-if ( ${CMAKE_SYSTEM_NAME} MATCHES Linux AND EXISTS /etc/os-release )
+-    execute_process(
+-        COMMAND grep -q alpine /etc/os-release
+-        RESULT_VARIABLE os_release_alpine
+-    )
+-
+-    if ( os_release_alpine EQUAL 0 )
++# Void-musl support
++if ( IS_VOID_MUSL )
```

Comment:
why not make this a generic var and even upstream that change?

Reply:
+1 Something like `NEED_LIBFTS` is way better then send that patch upstream.